### PR TITLE
M10Z Launch Post

### PR DESCRIPTION
**Pull request by LucaNerlich**
/schedule 2023-04-01

### CODE REVIEW

**Comment by LucaNerlich** [comment]
:hourglass: **Merge Schedule**
Scheduled to be merged on 2023-04-01 00:00:00 (UTC)

File: blog/articles/2023-04-01-welcome-to-m10z.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Hast du Freude daran die schrägsten Videospiele im Stream vorzustellen oder eine längere Kolumnenreihe zu Themen in der Videospielfinanzwelt zu
+verfassen…bitte melde Dich.
+
+Hast du Ideen zu passenden Themen, kannst sie aber selbst nicht ausformulieren oder brauchst Gegenspieler*innen in einem Podcast oder
+Livestream...bitte melde Dich.
+
+Das Ganze soll so niederschwellig wie möglich sein, probiere Dich aus.
+
+So das wirkt auf den ersten Blick jetzt sehr ausschweifend und wirft die Frage auf, wie diese ganzen Felder regelmäßig bespielt werden sollen. Und das
```
Comment (kkuez) [inline blog/articles/2023-04-01-welcome-to-m10z.md:62]: Ein Komma nach dem So.

File: blog/articles/2023-04-01-welcome-to-m10z.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Quo Vadis M10Z
+
+Jetzt ist der Bereich Videospiele oft Zeuge geworden, wie schlimm Neuausrichtungen nach hinten losgehen können (ja [Bomberman: Act Zero](https://en.wikipedia.org/wiki/Bomberman:_Act_Zero), Du bist
+gemeint). Es war es uns also wichtig einen anderen Ansatz zu finden. Dem Ganzen eine eigene Farbe zu geben (Orange wie es scheint, why not) und vor
+allem als Gemeinschaft. Was ihr jetzt vor Euch seht, ist noch nicht ganz das Ergebnis, aber der Weg zum Ziel. Ein offener Kanal, wie ihr es vielleicht
+vom Bürgerfunk kennt, mit dem Schwerpunkt Videospiele und alles Drumherum. Ein Portal für unsere und Eure Texte, Podcasts, Streams etc. Whatever
+floats your Jackdaw, wie die Assassinen sagen.
+
+Wenn ich von „unseren“ Inhalten spreche, meine ich den schon bestehenden, jetzt unter neuer Flagge / Farbe weitergeführten Community Podcast
+„Mindestens 10 Zeichen“, „Fundbüro“ der Forenmitglieder `Yollum` & `Lipardus` befasst sich mit achtlos zurückgelassenen Spielen und unser liebster
```
Comment (kkuez) [inline blog/articles/2023-04-01-welcome-to-m10z.md:43]: Vielleicht den "Mindestens 10 Zeichen"-Satz abschließen?

File: blog/articles/2023-04-01-welcome-to-m10z.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+verfassen…bitte melde Dich.
+
+Hast du Ideen zu passenden Themen, kannst sie aber selbst nicht ausformulieren oder brauchst Gegenspieler*innen in einem Podcast oder
+Livestream...bitte melde Dich.
+
+Das Ganze soll so niederschwellig wie möglich sein, probiere Dich aus.
+
+So das wirkt auf den ersten Blick jetzt sehr ausschweifend und wirft die Frage auf, wie diese ganzen Felder regelmäßig bespielt werden sollen. Und das
+ist der Kern des Ganzen. M10Z soll eine Spielwiese für uns und für Euch sein, auf der ihr Euch kreativ austoben könnt und dürft, aber niemals müsst.
+Wir haben nicht vor dies, auf welche Art auch immer, mit Geld und damit mit Erwartungen zu Verknüpfen. Sollte euch das Privatleben für einen Monat
```
Comment (kkuez) [inline blog/articles/2023-04-01-welcome-to-m10z.md:64]: verknüpfen

File: blog/articles/2023-04-01-welcome-to-m10z.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+einen Strich durch die Planung machen oder die Muse nicht küssen wollen, dann ist das halt so.
+Uns ist bewusst, dass es so zu objektiven Qualitätsschwankungen kommen kann, aber das fängt das Gesamtgefüge auf und ist meiner persönlichen Ansicht
```
Comment (kkuez) [inline blog/articles/2023-04-01-welcome-to-m10z.md:66]: Ein paar viele sos, vielleicht kann dieses hier weg?

File: blog/articles/2023-04-01-welcome-to-m10z.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+nach der Charme des Ganzen.
+
+## Und nun?
+
+Nun schaut euch doch in der nächsten Zeit öfter hier um, beteiligt euch an Threads zu den Artikeln im Wasted Forum oder beteiligt euch mit Inhalten an
```
Comment (kkuez) [inline blog/articles/2023-04-01-welcome-to-m10z.md:71]: Wasted-Forum

**Comment by LucaNerlich** [comment]
:x: **Merge Schedule**
Scheduled merge failed: Base branch was modified. Review and try the merge again.
In order to let the automerge-automation try again, the label "merge-schedule-failed" should be removed.

**Pull request merged**